search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt

```diff
@@ -62,7 +62,13 @@ class ListViewFilterChips @JvmOverloads constructor(
     private fun String.take20orDefault(): String {
         if (this.length <= 20)
```

Comment:
constant 20 should be in a variable

Reply:
it's default method name is take to get the string so I make for 20 character like this

Reply:
resolved